README.md

```diff
@@ -0,0 +1,52 @@
+<h1 align="center">
+<br />
+<a href="https://bloom-tronics.vercel.app/"><img src="./assets\BloomUI Homepage.jpg" alt="Bloom UI" width="300"></a>
```

Comment:
Also here, I guess you have given the wrong link. It should be for CL and not e-commerce right?